src/Uno.Toolkit.RuntimeTests/Tests/TabBarTests.cs

```diff
@@ -229,18 +229,17 @@ public async Task Verify_Indicator_Transitions(Orientation orientation, Indicato
 
 			var rootGrid = XamlHelper.LoadXaml<Grid>(@$"
```

Comment:
you can use $``` to avoid the "" and write like normal xaml (with the double-{} ofc)

Reply:
also are the new colors meant to be kept?

Reply:
oh woops no the colors should be removed :)